Makefile

```diff
@@ -47,6 +47,7 @@ unit-tests:
 	@bash ./scripts/unittests.sh
 
 # test runs ALL the test suites.
-test: systemtests unit-tests
+test:
+	@bash ./scripts/tests.sh
```

Comment:
If we do that, then that is inconsistent with the actual tests in both category (after this PR, they will all always run regardless of failures)

Do you run `make test` when writing code/tests or do you run the appropriate target directly?  I've always run the appropriate target so this wouldn't change anything for me in development.  I only run `make test` as a sanity test before committing